lib/twiml/TwiML.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+var builder = require('xmlbuilder');  /* jshint ignore:line */
+
+/* jshint ignore:start */
+/**
+ * Parent TwiML object
+ */
+/* jshint ignore:end */
+function TwiML() {
+  this.response = builder.create('Response').dec('1.0', 'UTF-8');
+}
+
+/* jshint ignore:start */
+/**
+ * Because child elements have properties named after their class names, we need
+ * to translate that when we want to access that at the parent prototype level.
+ * So this will translate something like Say to 'say' and VoiceResponse to
+ * 'response'.
+ */
+/* jshint ignore:end */
+TwiML.prototype._getPropertyName = function _getPropertyName() {
+  return this._propertyName === undefined ? 'response' : this._propertyName;
```

Comment:
is it hard to get the response classes to emit a `propertyName` as well? this seems like kind of a weird special case

Reply:
we can remove this logic now, right?

Reply:
Done.